code/modules/recycling/conveyor2.dm

```diff
@@ -458,4 +459,13 @@ GLOBAL_LIST_EMPTY(conveyors_by_id)
 	name = "paper- 'Nano-it-up U-build series, #9: Build your very own conveyor belt, in SPACE'"
 	info = "<h1>Congratulations!</h1><p>You are now the proud owner of the best conveyor set available for space mail order! We at Nano-it-up know you love to prepare your own structures without wasting time, so we have devised a special streamlined assembly procedure that puts all other mail-order products to shame!</p><p>Firstly, you need to link the conveyor switch assembly to each of the conveyor belt assemblies. After doing so, you simply need to install the belt assemblies onto the floor, et voila, belt built. Our special Nano-it-up smart switch will detected any linked assemblies as far as the eye can see! This convenience, you can only have it when you Nano-it-up. Stay nano!</p>"
 
+GLOBAL_LIST_INIT(conveyor_recipes, list (
+	new/datum/stack_recipe("conveyor filter", /obj/item/stack/conveyor_filter, 30), \
+	new/datum/stack_recipe("conveyor splitter", /obj/item/stack/conveyor_splitter, 30), \
+	))
+
+/obj/item/stack/conveyor/get_main_recipes()
+	. = ..()
+	. += GLOB.conveyor_recipes
+
```

Comment:
Why does this need to be a global list and not just a static one?

Reply:
to be honest, no clue. Just copied what the other stack recipies were doing. see:
https://github.com/Lobotomy-Corporation-13/lc13/blob/a0f1dc126ae02b195763e34eff47509dac7a3b01/code/game/objects/items/stacks/rods.dm#L40
https://github.com/Lobotomy-Corporation-13/lc13/blob/a0f1dc126ae02b195763e34eff47509dac7a3b01/code/game/objects/items/stacks/sheets/glass.dm#L44